static/app/utils/replays/getReplayEvent.spec.tsx

```diff
@@ -171,31 +181,51 @@ describe('getPrevReplayEvent', () => {
   it('should return undefined when the timestamp is earlier than any crumbs', () => {
     const crumbs = createCrumbs();
     const results = getPrevReplayEvent({
+      itemLookup: crumbs
+        .map(({timestamp}, i) => [+new Date(timestamp || ''), i])
+        .sort(([a], [b]) => a - b),
       items: crumbs,
       targetTimestampMs: START_TIMESTAMP_SEC * 1000 - CURRENT_TIME_MS,
     });
 
     expect(results).toBeUndefined();
   });
 
-  it('should return the crumb previous when a crumbs timestamp exactly matches', () => {
+  it('should return the last crumb if timestamp is later than any crumb', () => {
     const crumbs = createCrumbs();
-    const exactCrumbTime = 8135;
     const results = getPrevReplayEvent({
+      itemLookup: crumbs
+        .map(({timestamp}, i) => [+new Date(timestamp || ''), i])
+        .sort(([a], [b]) => a - b),
       items: crumbs,
-      targetTimestampMs: START_TIMESTAMP_SEC * 1000 + exactCrumbTime,
+      targetTimestampMs: 1652308892002 + 10,
+    });
+
+    expect(results?.id).toEqual(0);
```

Comment:
this `id=0` crumb seems like the first in the list. the title of the test made me think we should expect 166